test/test_meta.py

```diff
@@ -528,10 +527,12 @@ def run_meta_crossref(
             for a, ma in zip(flat_args, flat_meta_args):
                 flat_new_args.append(a if isinstance(a, torch.Tensor) and a.dtype in [torch.int8, torch.bool] else ma)
             meta_args = (meta_args[0], tree_unflatten(flat_new_args, spec))
-        elif func is torch.ops.aten.repeat_interleave.Tensor:
+        elif func in (torch.ops.aten.repeat_interleave.Tensor, torch.ops.aten.repeat_interleave.Tensor_out):
```

Comment:
This function is `run_meta_crossref`. IIRC there is a test that runs meta tests over the entire PyTorch codebase, not just over OpInfos. Do your changes impact that test?

Reply:
I'm only familiar with the test that runs over OpInfos, can you please share the test that runs meta tests over the entire codebase?

Reply:
Sorry, I am wrong about this. There is no test that runs meta tests over the entire pytorch codebase